### ISSUE TITLE
Gradient Penalty code error in WGAN_GP 

### ISSUE THREAD

**Issue by jwc0906** [hwalsuklee/tensorflow-generative-model-collections#33]
at 113 line in WGAN_GP, I recommend changing the code

`alpha = tf.random_uniform(shape=self.inputs.get_shape(), minval=0.,maxval=1.)`
to
`alpha = tf.random_uniform(shape=[BATCH_SIZE,1,1,1], minval=0.,maxval=1.)`

Because It  must be created **one** alpha value for each batch

**Comment by AliceAria** [comment]
You are right. I have the same view.The performance improved after changing thie code.

**Comment by iskangkang** [comment]
> You are right. I have the same view.The performance improved after changing thie code.

I have a question that wheather the bn in Discrimator should be removed?
